scripts/deploy.sh

```
@@ -0,0 +1,107 @@
+    echo "📊 Application status:"
+    pm2 status
+else
+    echo "❌ Health check failed!"
+    echo "🔄 Attempting rollback..."
+    
+    # Find latest backup
+    LATEST_BACKUP=$(ls -t $BACKUP_DIR/backup-* | head -n1)
+    
+    if [ -n "$LATEST_BACKUP" ]; then
+        echo "🔄 Rolling back to: $LATEST_BACKUP"
+        rm -rf $CURRENT_DIR/*
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)